### PR TITLE
✨ Add raw results for Token-Permissions

### PR DESCRIPTION
**Pull request by laurentsimon**
Add raw results for Token-Permissions

```release-notes
Add raw results for Token-Permissions
```

### CODE REVIEW

**Comment by laurentsimon** [comment]
Ready for review.

File: checker/raw_result.go [hunk 1]
```diff
@@ -243,3 +244,46 @@ type WorkflowJob struct {
...
+	// PermissionTypeWrite is a permission set to `write` for a permission we consider potentially dangerous.
+	PermissionTypeWrite PermissionType = "write"
+	// PermissionTypeRead is a permission set to `read`.
+	PermissionTypeRead PermissionType = "read"
+	// PermissionTypeNone is a permission set to `none`.
+	PermissionTypeNone PermissionType = "none"
+	// PermissionTypeUnknown is for other kinds of alerts, mostly to support debug messages.
+	// TODO: remove it once we have implemented severity (#1874).
+	PermissionTypeUnknown PermissionType = "unknown"
+)
```
Comment (azeemshaikh38) [inline checker/raw_result.go:278]: Shouldn't this be `PermissionLevel` and `PermissionType` should instead be - "content", "status" etc. ?

**Review by azeemshaikh38 — APPROVED** [review]
Overall, LGTM. Left a comment and there are also some CI failures. Please fix before submitting. Thanks!

File: checker/raw_result.go (hunk 1 above)
Comment (laurentsimon) [reply checker/raw_result.go:278]: thanks, will rename!

**Comment by laurentsimon** [comment]
> Overall, LGTM. Left a comment and there are also some CI failures. Please fix before submitting. Thanks!

Thanks. I will definitely will fix the CI

**Pull request merged**